### DISCUSSION TITLE
Arabic translations for daisyui

### DISCUSSION THREAD

**Discussion by souhaib-benbouzid** [saadeghi/daisyui#2471] [Ideas / feature requests]
Hey **daisyui** team,

I see that **daisyui** is available in multiple languages and I would love to see it also in my native language **Arabic**. 

I'm willing to work on the translations if you allow me to do so. 

Thank you.

**Comment by saadeghi** [comment]
That would be great!  
We're going to add automatic RTL mode in daisyUI 4.0 and also Document site. Arabic translation in RTL would be a nice addition.

The contribution guide is here: https://github.com/saadeghi/daisyui/blob/master/.github/CONTRIBUTING.md

The language files are here: https://github.com/saadeghi/daisyui/tree/master/src/docs/src/translation

You can copy `en.json`, rename it to `ar.json` and start translating.  
You don't need to translate all the strings at once to open a PR. it can be completed over time.


Let me know if you have any questions.

**Reply by souhaib-benbouzid** [reply]
🎉✨Awesome ✨🎉,  I will create a new issue and draft a PR for that.

**Reply by HichOoMa** [reply]
@souhaib-benbouzid I can participate

**Reply by souhaib-benbouzid** [reply]
@HaithamBH  sure, if you can go through the translations and check if there are any mistakes that would be great. 👍🏻
